internal/proftest/proftest.go

```diff
@@ -74,9 +74,10 @@ func EncodeJSON(x interface{}) []byte {
 // TestUI implements the plugin.UI interface, triggering test failures
 // if more than Ignore errors not matching IgnoreRx are printed.
 type TestUI struct {
-	T        *testing.T
-	Ignore   int
-	IgnoreRx string
+	T               *testing.T
+	Ignore          int
+	IgnoreRx        string
+	IgnoredErrCount int
```

Comment:
Maybe just IgnoreCount to be consistent as we don't include the "Err" string in Ignore and IgnoreRx names.